src/init.c

```diff
@@ -42,7 +42,7 @@ static const R_CallMethodDef CallEntries[] = {
     {"antsMotionCorrStats",             (DL_FUNC) &antsMotionCorrStats,              4},
     {"centerOfMass",                    (DL_FUNC) &centerOfMass,                     1},
     {"confidenceConnectedImage",        (DL_FUNC) &confidenceConnectedImage,         6},
-    {"connectedThresholdImage",         (DL_FUNC) &connectedThresholdImage,          5},
+    // {"connectedThresholdImage",         (DL_FUNC) &connectedThresholdImage,          5},
```

Comment:
shouldnt this be uncommented?

Reply:
No - I was adding both `confidenceConnectedImage` and `connectedThresholdImage` for the PR, but it was errroring so I was doing one at a time.  

I believe I did exactly what @ntustison https://github.com/ANTsX/ANTsR/pull/213/files had done, but I'm unsure why I'd get an error.  Do you think it has anything to do with `include` headers compared to https://github.com/ANTsX/ANTsR/pull/213/files#diff-b28a056446d4b76f1115cd7ec663a226?

Reply:
Still getting the error at https://travis-ci.org/ANTsX/ANTsR/jobs/394584887#L2193